devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1715,6 +1725,9 @@ bool WholeBodyDynamicsDevice::attachAllFTs(const PolyDriverList& p)
         ftSensors[IDTsensIdx] = ftList[deviceThatHasTheSameNameOfTheSensor];
     }
 
+    if (settings.disableSensorReadCheckAtStartup) {
+        return true;
+    }
```

Comment:
To be honest I am not completely convinced by the choice of exiting early. I would rather prefer to simply skip the part related to the startup check.

This could easily give us some headaches in the future if we simply add parts at the end of the method without noticing the ``return true``.